website/blog/2024-10-28-release-1.14.md

```diff
@@ -0,0 +1,223 @@
+---
```

Comment:
Might need update the name file so it the latest blog article ?

![image](https://github.com/user-attachments/assets/7eec4124-15d4-4cab-9da5-c135174af850)

Reply:
This should be sorted by date or am I wrong? 

This post has date of creation to 28th, and Introduction... is 29th => will the date change based on time of merging? / or specify the date if the time wont change when merged e.g. specifying `date: 2024-10-31`

Reply:
Nevermind the date is specified in the name of the file, now I see :D

Reply:
You will have to change the file name for it to reflect it!

So change the file name before you merge it in (ex 2024-10-31-release-1.14.md)

Reply:
Just commited :D